lisa/sut_orchestrator/ready.py

```diff
@@ -14,11 +17,25 @@
 from . import READY
 
 
+@dataclass_json()
+@dataclass
+class ReadyPlatformSchema:
+    # When set to True, the environment will not be deleted even if it becomes dirty.
+    reuse_dirty_env: bool = field(default=False)
```

Comment:
I set it as false by default, since when default value is set as true, the run against hello_world.yml which use local node type, will enter into dead loop. I didn't figure out the root cause yet.

Reply:
Do you mean, the case failed only the "default" is True, but not failed, if it's "set" to True?

Reply:
Please try to set the env status back to prepared, instead of deployed.